WeatherForecastController.cls

```diff
@@ -0,0 +1,139 @@
+        public Double temp;
+        public Integer humidity;
+        public Double feelsLike;
+        public String description;
+        public String icon;
+        public WrapperData(FromJSON parseData) {
+            Main te = parseData.main;
+            Weather we = parseData.weather[0];
+            this.temp = te.temp;
+            this.humidity = te.humidity;
+            this.feelsLike = te.feels_like;
+            this.description = we.description;
+            this.icon ='http://openweathermap.org/img/wn/' + we.icon + '@2x.png';
+        }
+    }
+    /******************************************************************
+     * @Name         : FromJSON
+     * @Description  : Contains objects of weather data
+     * @Created By   : Cagri Kilic
+     * @Created Date : Jan 17, 2023
+     * @Modification Log :
+     ******************************************************************
+     * Version        Developer        Date        Description
+     *------------------------------------------------------------
+     *
+     ******************************************************************/
+    public class FromJSON {
+        public Weather[] weather;
+        public Main main;
+    }
+    /******************************************************************
+     * @Name         : Weather
+     * @Description  : Contains variables of json data of weather
+     * @Created By   : Cagri Kilic
+     * @Created Date : Jan 17, 2023
+     * @Modification Log :
+     ******************************************************************
+     * Version        Developer        Date        Description
+     *------------------------------------------------------------
+     *
+     ******************************************************************/
+    public class Weather {
+        public String description;
+        public String icon;
+    }
+    /******************************************************************
+     * @Name         : Main
+     * @Description  : Contains variables of json data of main
+     * @Created By   : Cagri Kilic
+     * @Created Date : Jan 17, 2023
+     * @Modification Log :
+     ******************************************************************
+     * Version        Developer        Date        Description
+     *------------------------------------------------------------
+     *
+     ******************************************************************/
+    public class Main {
+        public Double temp;
+        public Double feels_like;
+        public Integer humidity;
+    }
+    /******************************************************************
+  	* @Name         : parse 
+  	* @Description  : Parses json data
+  	* @Created By   : CagriKilic
+  	* @Created Date : Jan 17, 2023
+    * @Param json   : String - Data of Http response
+    * @Return       : FromJSON - Weather datas
+  	******************************************************************/
+    private static FromJSON parse(String json) {
```

Comment:
```suggestion
    private static fromJSON parse(String json) {
```

Reply:
FromJSON is a class. I know we have to write the class names in pascal. 
My method name is 'parse'.